definitions/checks/foreman/check_https_proxies.rb

```diff
@@ -1,34 +0,0 @@
-module Checks
-  class CheckHttpsProxies < ForemanMaintain::Check
-    metadata do
-      label :https_proxies
-      for_feature :foreman_database
-      description 'Check for HTTPS proxies from the database'
-      manual_detection
-    end
-
-    def run
-      https_proxies = find_https_proxies
-      unless https_proxies.empty?
-        https_proxy_names = https_proxies.map { |proxy| proxy['name'] }
-        question = "Syncing repositories through an 'HTTP Proxy' that uses the HTTPS\n"\
-                    "protocol is not supported directly with Satellite 6.10.\n"\
```

Comment:
technically, that BZ is still open and unsolved, so I would keep that check?

Reply:
(but it's `manual_detection` and the last use was in `definitions/scenarios/upgrade_to_satellite_6_10.rb` :woman_shrugging: )

Reply:
Linkie to the BZ?

Reply:
Hrmm, yea so when upgrading to 6.10 it was an issue, and there is nothing about the "upgrade" itself that would cause this to be better or worse. I think for upgrades this check does not make sense anymore. The question then is is this worth a health check? I would expect users to be broken if they tried this at any given time and then they would revert the change.

Reply:
probably, yeah

nuke it